suite-common/graph/src/hooks.ts

```diff
@@ -36,6 +36,31 @@ type CommonUseGraphReturnType = {
 // if start date is null we are fetching all data till first account movement
 type StartOfTimeFrameDate = Date | null;
 
+const MINIMAL_PROPORTIONAL_EDGE_OFFSET = 0.02;
```

Comment:
How is this number determined? Trial and error or is there a reason for this? Could you please attach a comment if there is any logic behind it?

Reply:
comment added in cb5b45438